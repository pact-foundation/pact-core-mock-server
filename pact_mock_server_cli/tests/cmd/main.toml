bin.name = "pact-mock-server"
args = "--help"
#status.code = 2
