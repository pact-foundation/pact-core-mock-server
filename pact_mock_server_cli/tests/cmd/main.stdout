Standalone Pact mock server

Usage: pact-mock-server [OPTIONS] [COMMAND]

Commands:
  start            Starts the master mock server
  list             Lists all the running mock servers
  create           Creates a new mock server from a pact file
  verify           Verify the mock server by id or port number, and generate a pact file if all ok
  shutdown         Shutdown the mock server by id or port number, releasing all its resources
  shutdown-master  Performs a graceful shutdown of the master server (displayed when it started)
  help             Print this message or the help of the given subcommand(s)

Options:
      --help                 Print help and exit
  -v, --version              Print version information and exit
  -p, --port <port>          port the master mock server runs on (defaults to 8080)
  -h, --host <host>          hostname the master mock server runs on (defaults to localhost)
  -l, --loglevel <loglevel>  Log level for mock servers to write to the log file (defaults to info) [possible values: error, warn, info, debug, trace, none]
      --no-term-log          Turns off using terminal ANSI escape codes
      --no-file-log          Do not log to an output file

